pallets/ddc-clusters/src/lib.rs

```diff
@@ -247,6 +246,27 @@ pub mod pallet {
 		}
 	}
 
+	impl<T: Config> Pallet<T> {
+		pub fn do_create_clusters(
```

Comment:
we're creating 1 cluster here and this is just an internal method, so:
1. rename to do_create_cluster not do_create_cluster**s**. 
2. better make it private (remove pub).